Week3/homework/step2-2.js

```diff
@@ -2,19 +2,29 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
+  for (let i = startIndex; i < stopIndex + 1; i++) {
+    numbers.push(i);
+  }
+  for (const value of numbers) {
+    if (value % 3 === 0) {
+      threeCallback(value);
+      if (value % 5 === 0) {
+        fiveCallback(value);
+      }
+    }
+    if (value % 5 === 0 && !(value % 3 === 0)) {
+      fiveCallback(value);
+    }
+  }
   console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
```

Comment:
This console.log was supposed to be replaced by your code (i.e. deleted).

Reply:
removed.